$OpenBSD: patch-src_libpiano_piano_h,v 1.3 2012/05/28 09:47:00 dcoppa Exp $

Fix ambiguous error message "Invalid Partner Login" on wrong
email address/password failure
(upstream git commit 81bf363ac69e0036562434ca0e7153c3e624c8c3)

--- src/libpiano/piano.h.orig	Sun May  6 16:33:54 2012
+++ src/libpiano/piano.h	Mon May 28 10:59:07 2012
@@ -244,6 +244,7 @@ typedef enum {
 	PIANO_RET_INVALID_RESPONSE = 2,
 	PIANO_RET_CONTINUE_REQUEST = 3,
 	PIANO_RET_OUT_OF_MEMORY = 4,
+	PIANO_RET_INVALID_LOGIN = 5,
 
 	PIANO_RET_P_INTERNAL = PIANO_RET_OFFSET+0,
 	PIANO_RET_P_API_VERSION_NOT_SUPPORTED = PIANO_RET_OFFSET+11,
